nose_ignores.txt

```diff
@@ -37,3 +37,7 @@
 --ignore-file=test_gadget_pytest\.py
 --ignore-file=test_vr_orientation\.py
 --ignore-file=test_particle_trajectories_pytest\.py
+--ignore-file=test_image_array\.py
```

Comment:
note to reviewers:
I've added modules that are now pytest-only due to usage of `teardown_module`. Many other modules have a `setup_module` method bu no `teardown_method`, and what they effectively do in setups seems safe (as in, free of side effects), and, to the best of my knowledge, is not actually used in nose.
```python
def setup_module():
    from yt.config import ytcfg

    ytcfg["yt", "internals", "within_testing"] = True
```

Reply:
Thanks, that makes sense.  Do you think that having pytest-only modules will change the relative runtimes of the different test suites?

Reply:
not in any significant way. The four modules I just made pytest-only (combined) only take a couple seconds to run locally.